File: Android_Zone_Test/src/com/example/mylib_test/activity/system/ResultActivity3.java
package com.example.mylib_test.activity.system;

import com.example.mylib_test.R;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.view.View.OnClickListener;

public class ResultActivity3 extends Activity implements OnClickListener {
	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_result3);
	}

	@Override
	public void onClick(View v) {
		switch (v.getId()) {
		case R.id.bt_returnOne:
			setResult(SystemMainActivity.ResponseCode);
			finish();
			break;

		default:
			break;
		}
	}

}

File: Android_Zone_Test/src/com/example/mylib_test/activity/system/ResultActivity3.kt
package com.example.mylib_test.activity.system

import com.example.mylib_test.R

import android.view.View
import com.zone.lib.base.controller.activity.BaseFeatureActivity

class ResultActivity3 : BaseFeatureActivity() {
    override fun setContentView() {
        setContentView(R.layout.activity_result3)
    }

    override fun initData() {
    }

    override fun setListener() {
    }


    override fun onClick(v: View?) {
        when (v?.id) {
            R.id.bt_returnOne -> {
                setResult(SystemMainActivity.ResponseCode)
                finish()
            }
            else -> {
            }
        }
    }

}
